src/client.ts

```diff
@@ -2051,6 +2083,44 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         });
     }
 
+    /**
+     * Initialise support for end-to-end encryption in this client, using the rust matrix-sdk-crypto.
+     *
+     * An alternative to {@link initCrypto}.
+     *
+     * *WARNING*: this API is very experimental and should not yet be used in production code! Eventually
+     *    it will be deprecated and `initCrypto` will do the same thing.
```

Comment:
```suggestion
     * *WARNING*: this API is very experimental, should not be used in production, and may change without notice!
     *    Eventually it will be deprecated and `initCrypto` will do the same thing.
```

just an added note that we're not going to declare API changes as breaking, as some people might assume experimental==safe enough to use (unfortunately).